ssc/core.cpp

```diff
@@ -125,6 +125,7 @@ bool compute_module::evaluate()
                 return false;
             }
         }
+	return true;
```

Comment:
I do not think we want this line there- we want this function not to return anything if it doesn't return false. @sjanzou can you confirm?

Reply:
@janinefreeman, the "return true;" actually should be there since the CallSscEquations is a Lambda function with an implicit boolean type that should return a boolean from all control paths. However, this will not make a difference in the results since the return value is not used anywhere... lines 130 and 144.

Currently, the function returns 1>C:\Projects\GitHub\NREL\develop\ssc\ssc\core.cpp(128): warning C4715: '<lambda_c23638134389f859b3dd7a343978d202>::operator()': not all control paths return a value

Reply:
Ok, thank you for clarifying @sjanzou !